File: 13Feb2024.java
class Solution {
    public String firstPalindrome(String[] words) {
        for (String s : words) {
            boolean isValid = true;
            for (int i = 0, j = s.length() - 1; i < j; i++, j--) {
                if (s.charAt(i) != s.charAt(j)) {
                    isValid = false;
                }
            }
            if (isValid)
                return s;
        }
        return "";
    }
}

File: 13Feb2024.py
from ast import List


class Solution:
    def firstPalindrome(self, words: List[str]) -> str:
        for s in words:
            is_valid = True
            for i in range(len(s)//2):
                if s[i] != s[len(s)-1-i]:
                    is_valid = False
                    break
            if is_valid:
                return s
        return ""